README.md

```diff
@@ -22,6 +22,8 @@ export CONFIG_MAP_BOOSTER_VALUE="Hello, %s from a ConfigMap !"
 
 export SECURED_HTTP_BOOSTER_PASSWORD=password
 export SECURED_HTTP_BOOSTER_USERNAME=alice
+
+export BOOSTER_RUNTIME="vertx" //It is possible to set vertx,nodejs,springboot,wfswarm
```

Comment:
Can't we infer that from the booster name? The less parameters we need to set, the better.

Reply:
Actually ignore that -- it seems we have the tests set up the way I thought we do. We'll have to change that, but not in this PR.

Reply:
I haven't any information about booster name. Probably I can infer it from booster's URL